app/main.py

```diff
@@ -1,5 +1,71 @@
+from typing import List, Optional
+
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: Optional[List[int]] = None
+                 ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+        self,
+        name: str,
+        weight: int,
+        coords: Optional[List[int]] = None
+    ) -> None:
+        super().__init__(name, weight, coords)
+        self.coords = coords if coords is not None else [0, 0, 0]
```

Comment:
Do no not redefine `self.coords` after the initialization in the parent class, define it while passing values to the `super().__init__(`
```suggestion
        super().__init__(name, weight, coords or [0, 0, 0])
```